addons/mass_mailing/tests/test_mailing_controllers.py

```diff
@@ -73,22 +73,24 @@ def test_tracking_url_token(self):
             mailing.action_send_mail()
 
         mail = self._find_mail_mail_wrecord(self.test_contact)
+        mail_id_int = mail.id
+        mail_tracking_url = mail._get_tracking_url()
         mailing_trace = mail.mailing_trace_ids
         self.assertEqual(mail.state, 'sent')
         self.assertEqual(len(mailing_trace), 1)
         self.assertFalse(mailing_trace.open_datetime)
         self.assertEqual(mailing_trace.trace_status, 'sent')
+        mail.unlink()  # the mail might be removed during the email sending
```

Comment:
Shoudln't we flush to be sure it is removed ? Not sure it impacts the test, thinking of it.